app/uk/gov/hmrc/cdsimportsddsfrontend/views/components/inputs/input_text.scala.html

```diff
@@ -23,7 +23,7 @@
   value: Option[String] = None
 )(implicit messages: Messages)
 <div class="govuk-form-group">
-   <label class="govuk-label" for="@{field.id}">@messages(label, labelArgs: _*)</label>
+   <label class="govuk-label" for="@{field.id}">@Html(messages(label, labelArgs: _*))</label>
```

Comment:
i don't think we should be putting html in the messages file - is there another way to achieve the layout you're after?

Reply:
+1

Reply:
normally I would try to achieve it with css but it's kind of a temporary solution just to make input fields in one line in our temporary big form solution. Is there any big downside to this solution?

Reply:
fixed